pkg/broker/options.go

```diff
@@ -24,4 +24,6 @@ type Options struct {
 	// The default namespace wheer Minibroker deploys service instances.
 	DefaultNamespace          string
 	ServiceCatalogEnabledOnly bool
+	// The YAML file where the optional default chart values are stored.
+	OverrideChartParams string
```

Comment:
Maybe call this `OverrideChartParamsPath`? So it's clear in the code which is which.